packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/restore/constants.js

```diff
@@ -0,0 +1,26 @@
+export const SNAPSHOT_TYPE = {
+  AUTOMATIC: 'automatic',
+  MANUAL: 'manual',
+  SYSTEM: 'system'
+}
+
+export const SNAPSHOT_STATUS = {
+  AVAILABLE: 'available',
+  CREATING: 'creating',
+  DELETING: 'deleting',
+  ERROR: 'error',
+  ERROR_DELETING: 'error_deleting',
+  MANAGE_ERROR: 'manage_error',
+  MANAGE_STARTING: 'manage_starting',
+  RESTORING: 'restoring',
+  UNMANAGE_ERROR: 'unmanage_error',
+  UNMANAGE_STARTING: 'unmanage_starting'
+}
+
+export const FETCH_INTERVAL = 5000;
```

Comment:
I dont see the usage of this constant. Not needed as of now ?

Reply:
it's WIP it will be used or remove in the next PR